src/test/regress/Makefile

```diff
@@ -47,6 +47,8 @@ check-worker: all
 
 check-multi: all tempinstall-main
 	$(pg_regress_multi_check) --load-extension=citus \
+	--server-option=citus.task_tracker_delay=10ms \
+	--server-option=citus.remote_task_check_interval=5ms \
```

Comment:
1- added these settings to pg_regress_multi.pl
2- further reduced remote_task_check_interval to 1ms. reducing task tracker delay actually increased running time.